roles/reverseproxy/files/Caddyfile

```diff
@@ -0,0 +1,7 @@
+chat.spaceapi.net {
+  proxy / localhost:8200 {
```

Comment:
The port *could* be extracted to a variable, but it's not that important.

Reply:
Next time, my lazy ass doesn't want to make a template out of it right now, the moment there will be more it would be smart to do so :)